glean-core/ios/Glean/Glean.swift

```diff
@@ -66,7 +67,7 @@ public class Glean {
         self.configuration = configuration
 
         self.handle = withFfiConfiguration(
-            dataDir: getDocumentsDirectory().absoluteString,
+            dataDir: getDocumentsDirectory().relativePath,
```

Comment:
Why does this need to be relative?

Reply:
Explained in the commit message:

> We handle file URLs only. These have a 'file://' prefix.
> We don't want the 'file://' prefix.

I don't really have an idea why it worked before.

Reply:
We handle as in.. *iOS* requires us to?

Reply:
The iOS API we use (`FileManager.default.urls`) gives us URLs.
But we need to pass these through to glean-core, so turn them into strings.
And then the low-level Rust APIs expect proper path strings, not file URLs.

Reply:
I see, thanks for clarifying. Can you add a comment with what you just wrote before this line?